kuttl-test.yaml

```diff
@@ -0,0 +1,23 @@
+#
+# EXECUTION (from install_yamls repo root):
+#
+#   make keystone_kuttl
+#
+# ASSUMPTIONS:
+#
+# 1. Latest version of kuttl is installed at /usr/local/bin/kubectl-kuttl
+#    - wget https://github.com/kudobuilder/kuttl/releases/download/v0.11.1/kubectl-kuttl_0.11.1_linux_x86_64
+#    - mv kubectl-kuttl_0.11.1_linux_x86_64 /usr/local/bin/kubectl-kuttl
+#    - chmod 755 /usr/local/bin/kubectl-kuttl
+# 2. An OCP 4.10+ CRC cluster with Podified Operators has been deployed
+# 3. CLI user has access to $KUBECONFIG
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
+reportFormat: JSON
+reportName: kuttl-test-horizon
+timeout: 180
+namespace: horizon-kuttl-tests
```

Comment:
For the moment, let's switch this namespace to `openstack`, as that is what is used elsewhere by the tests (otherwise the current test will fail).  We can follow-up later when we sweep through operators to support arbitrary-namespace KUTTL tests.